probes/blocksForcePushOnBranches/impl.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2023 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//nolint:stylecheck
+package blocksForcePushOnBranches
+
+import (
+	"embed"
+	"fmt"
+
+	"github.com/ossf/scorecard/v4/checker"
+	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes/internal/utils/uerror"
+)
+
+//go:embed *.yml
+var fs embed.FS
+
+const Probe = "blocksForcePushOnBranches"
+
+func Run(raw *checker.RawResults) ([]finding.Finding, string, error) {
+	if raw == nil {
+		return nil, "", fmt.Errorf("%w: raw", uerror.ErrNil)
+	}
+
+	r := raw.BranchProtectionResults
+	var findings []finding.Finding
+
+	for i := range r.Branches {
+		branch := &r.Branches[i]
+		if branch.BranchProtectionRule.AllowForcePushes != nil {
+			switch *branch.BranchProtectionRule.AllowForcePushes {
```

Comment:
similar comment on repetition. see blocksDelete for more.

Reply:
https://github.com/ossf/scorecard/pull/3691/commits/99b9290aaafa287a08b9281c1f191af33afea69b